src/main/scala/com/cloudera/sparkts/Frequency.scala

```diff
@@ -95,10 +85,9 @@ class BusinessDayFrequency(val days: Int) extends Frequency {
   }
 
   override def equals(other: Any): Boolean = {
-    if (other.isInstanceOf[BusinessDayFrequency]) {
```

Comment:
I think that's what it compiles to... roughly:

```
[INFO]     override def equals(other: Object): Boolean = {
[INFO]       case <synthetic> val x1: Object = other;
[INFO]       case5(){
[INFO]         if (x1.$isInstanceOf[com.cloudera.sparkts.BusinessDayFrequency]())
[INFO]           {
[INFO]             <synthetic> val x2: com.cloudera.sparkts.BusinessDayFrequency = (x1.$asInstanceOf[com.cloudera.sparkts.BusinessDayFrequency](): com.cloudera.sparkts.BusinessDayFrequency);
[INFO]             matchEnd4(x2.days().==(BusinessDayFrequency.this.days()))
[INFO]           }
[INFO]         else
[INFO]           case6()
[INFO]       };
[INFO]       case6(){
[INFO]         matchEnd4(false)
[INFO]       };
[INFO]       matchEnd4(x: Boolean){
[INFO]         x
[INFO]       }
[INFO]     };
```

convincing? or switch it back?